project/plugins.sbt

```diff
@@ -1,10 +1,13 @@
 resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-accessibility-linter" % "0.33.0")
-addSbtPlugin("uk.gov.hmrc" %% "sbt-auto-build" % "3.16.0") //cannot go any higher on Scala 2.12
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
-addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "1.9.3")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.18")
-addSbtPlugin("io.github.irundaia" % "sbt-sassify" % "1.5.2")
\ No newline at end of file
+// To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
+ThisBuild / libraryDependencySchemes ++= Seq("org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always)
```

Comment:
is this needed ?

Reply:
good catch it's no longer required

Reply:
apparently it's still needed https://build.tax.service.gov.uk/job/BTA/job/VAT%20Registration/job/vat-registration-eligibility-frontend-pr-builder/58/console

Reply:
but for some reason it works locally